README.md

```diff
@@ -1,39 +1,15 @@
-# Terraform Provider Astro
+<div align="center">
+    <picture>
+        <img src="https://github.com/user-attachments/assets/22586f12-3871-4bb6-8ead-40bec82ec3ce" width="200">
+    </picture>
+    <p>Official Astro Terraform Provider to configure and manage your Astro infrastructure through an API.</p>
+    <a href="https://registry.terraform.io/providers/astronomer/astro/latest/docs"><img src="https://img.shields.io/static/v1?label=Docs&labelColor=0F0C27&message=terraform-provider-astro&color=4E408D&style=for-the-badge" /></a>
+    <a href="https://astronomer.docs.buildwithfern.com/docs/api/overview"><img src="https://img.shields.io/static/v1?label=Docs&labelColor=0F0C27&message=API Ref&color=4E408D&style=for-the-badge" /></a>
+</div>
 
-## Requirements
-
-- [Terraform](https://developer.hashicorp.com/terraform/downloads) >= 1.7
-- [Go](https://golang.org/doc/install) >= 1.21
-
-## Building The Provider
-
-1. Clone the repository
-2. Enter the repository directory
-3. Build the provider using the following `Makefile` command:
-
-```shell
-make dep
-make build
-```
-
-4. The provider binary will be available in the `bin` directory
-
-## Adding Dependencies
-
-This provider uses [Go modules](https://github.com/golang/go/wiki/Modules).
-Please see the Go documentation for the most up-to-date information about using Go modules.
-
-To add a new dependency `github.com/author/dependency` to your Terraform provider:
-
-```shell
-go get github.com/author/dependency
-go mod tidy
-```
-
-Then commit the changes to `go.mod` and `go.sum`.
 
 ## Using the provider
```

Comment:
i think this section is missing information. so far it only includes information on how to get started but not how to use the provider. how does the user know how to create a resource? or how to create a data source? or how to output values? 

i think the solution is to include information from other guides in a condensed form, and at least cover what types of resources/data sources are supported. can also include links to existing guides

Reply:
Thank you Isaac, I added a link to our full registry docs which include supported resources and data sources along with more detailed usage instructions.